waku/v2/node/storage/sqlite.nim

```diff
@@ -260,6 +261,9 @@ proc migrate*(db: SqliteDatabase, path: string, targetVersion: int64 = migration
       return err("failed to filter migration scripts")
     
     let scripts = scriptsRes.value
+    if (scripts.len == 0):
```

Comment:
Sorry for being late to this PR, could not review it yesterday. Thanks Hanno for addressing the problem.

As we spoke in the call, maybe a more proper check is to see whether there is at least one script for all the versions that fall between `userVersion` and `targetVersion` (excluding `userVersion`), we will probably need a helper function to check this. This also implies that `scripts.len` should be at least `targetVersion-userVersion`.
Having this said, since this PR is particularly to address the fleet issue, maybe the current check `scripts.len == 0` is sufficient.